tests/bluetooth/tester/src/btp_csis.c

```diff
@@ -61,6 +63,18 @@ static uint8_t csis_get_member_rsi(const void *cmd, uint16_t cmd_len,
 	return BTP_STATUS_VAL(err);
 }
 
+static uint8_t csis_set_sirk_type(const void *cmd, uint16_t cmd_len, void *rsp,
+				  uint16_t *rsp_len)
+{
+	const struct btp_csis_set_sirk_type_cmd *cp = cmd;
+
+	enc_sirk = cp->encrypted;
```

Comment:
```suggestion
	enc_sirk = cp->encrypted != 0U;
```

Suggest proper conversion from `uint8_t` to `bool`

Reply:
done